C++/source/searching/jump_search.cpp

```diff
@@ -0,0 +1,54 @@
+// C++ program to implement Jump Search
```

Comment:
Please read the [contribution guidelines](https://github.com/faheel/Algos/blob/master/CONTRIBUTING.md) to see how to format this comment.